src/Components/Blazor/WebAssembly.Authentication/src/DefaultAuthenticationManager.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Components.WebAssembly.Authentication
+{
+    /// <summary>
+    /// An <see cref="AuthenticationManager{TAuthenticationState}"/> that uses <see cref="RemoteAuthenticationState"/> as the
+    /// state to be persisted across authentication operations.
+    /// </summary>
+    public class DefaultAuthenticationManager : AuthenticationManager<RemoteAuthenticationState>
```

Comment:
Just to check, is it a limitation of the razor compiler that we're required to give this a different name instead of using generic type overloading?

(That is, what breaks if this was simply called `AuthenticationManager` instead? Just like both `List` and `List<T>` exist.)

Reply:
We changed this to RemoteAuthenticatorView